Dockerfile

```diff
@@ -69,23 +77,54 @@ RUN ${CONDA_EXECUTABLE} env create -f ./danesfield/deployment/conda/conda_env.ym
 
 # Install core3d-tf_ops package from kitware-danesfield / defaults
 RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
-      conda activate core3d && \
-      conda install -c kitware-danesfield -c kitware-danesfield-df -y core3d-tf_ops && \
-      conda clean -tipsy"]
+  conda activate core3d && \
+  pip install pip==20.0.1 && \
+  conda install -c kitware-danesfield -c kitware-danesfield-df -y core3d-tf_ops && \
+  conda clean -tipsy"]
 
 # Install opencv package from conda-forge
 RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
-      conda activate core3d && \
-      conda install -c conda-forge -y opencv && \
-      conda clean -tipsy"]
+  conda activate core3d && \
+  conda install -c conda-forge -y opencv && \
+  conda clean -tipsy"]
+
+# Copy patches for Colmap and VisSat
+COPY patches /patches
+
+# Install ColmapForVisSat from Github
+RUN git clone --recursive https://github.com/Kai-46/ColmapForVisSat.git && \
+  cd ColmapForVisSat && \
+  git checkout 9d96671 && \
+  git apply ../patches/colmap_deps.patch && \
+  chmod +x /ColmapForVisSat/ubuntu1804_install_dependencies.sh && \
+  chmod +x /ColmapForVisSat/ubuntu1804_install_colmap.sh && \
+  apt-get update && \
+  /ColmapForVisSat/ubuntu1804_install_dependencies.sh && \ 
+  cd /ColmapForVisSat && \
+  ./ubuntu1804_install_colmap.sh
+
+# Install VisSat package from Github
+RUN ["/bin/bash", "-c", "git clone https://github.com/Kai-46/VisSatSatelliteStereo.git && \
+  cd VisSatSatelliteStereo && \
+  git checkout e5ca3a0 && \
+  git apply ../patches/vissat.patch && \
+  source /opt/conda/etc/profile.d/conda.sh && \
+  conda activate core3d && \
+  pip install -r /VisSatSatelliteStereo/requirements.txt"]
+
+# Install LAStools package from Github
+RUN git clone https://github.com/LAStools/LAStools.git && \
+  cd LAStools && \
+  make
 
 # Install Danesfield package into CORE3D Conda environment
 COPY . ./danesfield
 RUN rm -rf ./danesfield/deployment
 RUN ["/bin/bash", "-c", "source /opt/conda/etc/profile.d/conda.sh && \
-      conda activate core3d && \
-      pip install --upgrade pip && \
-      pip install -e ./danesfield"]
+  conda activate core3d && \
+  pip install -e ./danesfield"]
+
+COPY ply2txt.py ply2txt.py
```

Comment:
This is the file you just removed right?  So we should also remove it here I think.